lib/sqlalchemy/orm/dynamic.py

```diff
@@ -102,18 +129,24 @@ class AppenderMixin(AbstractCollectionWriter[_T]):
 
     """
 
-    query_class = None
+    query_class: Type[Query[_T]] = None  # type: ignore[assignment]
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4507